homeassistant/components/camera/arlo.py

```diff
@@ -14,15 +14,30 @@
 from homeassistant.components.arlo import DEFAULT_BRAND, DATA_ARLO
 from homeassistant.components.camera import Camera, PLATFORM_SCHEMA
 from homeassistant.components.ffmpeg import DATA_FFMPEG
+from homeassistant.const import ATTR_BATTERY_LEVEL
 
 DEPENDENCIES = ['arlo', 'ffmpeg']
-
 _LOGGER = logging.getLogger(__name__)
 
+ATTR_BRIGHTNESS = 'brightness'
+ATTR_FLIPPED = 'flipped'
+ATTR_MIRROED = 'mirroed'
```

Comment:
Sorry, I only meant to point out the typo. It should be `ATTR_MIRRORED` and `'mirrored'`.